### DISCUSSION TITLE
Embed script locally and flash it on Technic Hub with pybricksdev

### DISCUSSION THREAD

**Discussion by Vinz1911** [pybricks/support#444] [Q&A] [answered]
Hello 👋 

With the Web Interface there is the possibility to add my script to the Firmware and flash it on the Technic Hub. Is this also possible to do that with pybricksdev? Can I add the .mpy script to the firmware locally and then flash it on the hub? 

Thank you and Best regards

**Comment by laurensvalk** [comment, marked as answer]
Yes. There's a Python file in the firmware zip file which you can change as you like. No need to tweak binaries manually.
